### ISSUE TITLE
`require.config` should reuse contexts of the same name

### ISSUE THREAD

**Issue by dgoldstein0** [requirejs/alameda#25]
In requirejs, if `require.config` is called with a `context` option matching an already existing context, that context is reused.  Alameda, however, creates a new context every time.

Namely:

```
require.config({context: "hi"}) === require.config({context: "hi"})
```

evaluates to `true` in requirejs but `false` in alameda.

**Comment by jrburke** [comment]
Thanks for the report. Fixed in 1.1.2, tagged and released.

**Comment by dgoldstein0** [comment]
thanks for the quick fix - this made my night, as I really wasn't expecting a response in less than a day or two.

One of my coworkers spent a few hours on a workaround (not yet finished) which I'm glad we'll no longer need.
